pmm/vcpkg.cmake

```diff
@@ -145,10 +145,41 @@ function(_pmm_vcpkg_default_triplet out)
     set(${out} "${arch}-${platform}" PARENT_SCOPE)
 endfunction()
 
+function(_pmm_vcpkg_copy_custom_ports ports_list)
+    foreach (port IN LISTS ports_list)
+        get_filename_component(port_name ${port} NAME)
+        if (NOT EXISTS "${port}/portfile.cmake")
+            message(FATAL_ERROR "Failed find portfile in ${port}!")
+        endif()
+        set(port_location "${vcpkg_inst_dir}/ports/${port_name}")
+        if(EXISTS "${port_location}" AND NOT EXISTS "${port_location}/CUSTOM_PORT_FROM_PMM.txt")
+          message(WARNING "Portfile already included by default!")
+        elseif(NOT EXISTS "${port_location}/CUSTOM_PORT_FROM_PMM.txt")
+          file(MAKE_DIRECTORY "${port_location}")
+          file(WRITE "${port_location}/CUSTOM_PORT_FROM_PMM.txt" "This is a custom port copied by PMM")
+        endif()
+        file(GLOB port_files "${port}/*")
+        foreach (port_file IN LISTS port_files)
+          get_filename_component(port_file_name ${port_file} NAME)
+          set(port_file_location "${port_location}/${port_file_name}")
+          file(TIMESTAMP ${port_file_location} port_file_location_ts)
+          file(TIMESTAMP ${port_file} port_file_ts)
+          _pmm_log(DEBUG "Timestamp: ${port_file_location_ts} for ${port_file_location}")
+          _pmm_log(DEBUG "Timestamp: ${port_file_ts} for ${port_file}")
+          if(NOT "${port_file_ts}" STREQUAL "${port_file_location_ts}")
```

Comment:
I just realized that there's an easier way to compare file timestamps. Rather than explicitly getting the timestamp like here, you can use `if(... IS_NEWER_THAN ...)`. See https://github.com/vector-of-bool/pmm/blob/3b4b94857d86cfa27976ce0785d29d11a752de83/pmm/conan.cmake#L444

Reply:
The issue with `IS_NEWER_THAN` is that if the file timestamps are exactly the same, the `IS_NEWER_THAN` comparison returns true

Reply:
True, but odds are you aren't going to have two files with identical timestamps. And if you do, it's not a problem to copy the file over, thus updating the timestamp.

Reply:
I've tried using `IS_NEWER_THAN` but it will always copy the files even when there aren't any changes